#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_changelog-minor-typo-fixes.dpatch by Anthony Fok <foka@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Minor typo fixes to upstream ChangeLog.

@DPATCH@

--- stardict-3.0.1.orig/ChangeLog
+++ stardict-3.0.1/ChangeLog
@@ -12,7 +12,7 @@
 
 2007.8.13
 StarDict-3.0.0 (RedHat) released.
-http://www.StarDict.org on-line dictionries website is out in 2006.9.25. We have developed the stardictd dictionary server for it.
+http://www.StarDict.org on-line dictionaries website is out in 2006.9.25. We have developed the stardictd dictionary server for it.
 StarDict's total download count reached two millions in 2007.3.8.
 Network dictionaries support.
 Plugin system.
@@ -38,14 +38,14 @@
 KingSoft PowerWord support. This added 158 dictionaries. The data is cracked by Dwing(http://free5.ys168.com/?dwing).
 Synonym support.
 Sort word list by collate function support.
-XDXF project(http://xdxf.sourceforge.net) has port its dictionaries to StarDict format, some dictionries are come from Sdictionary. This added 615 dictionries.
+XDXF project (http://xdxf.sourceforge.net) has port its dictionaries to StarDict format, some dictionaries are come from Sdictionary. This added 615 dictionaries.
 French translation, thanks Christophe Durville <cdurpp@ifrance.com>.
 Brazilian Portuguese translation, thanks Ernesto <ernesto_mor@yahoo.com.ar>.
 Add cfdict dictionary, thanks Gilles Vigner <xiaogil@yahoo.fr>.
 Add English-Thai Lexitron Dictionary and Thai-English Lexitron Dictionary, thanks Suphanut Thanyaboon <suphanut@tcllab.org>.
 Add many Japanese dictionaries.
 Add some Chinese dictionaries, data comes from KDic.
-Add two Germany dictionries: ldaf, xindehan.
+Add two Germany dictionaries: ldaf, xindehan.
 Add duden Germany dictionary, thanks Liu Guizhu <zhuzi813@gmail.com> for providing data file.
 Add fundset dictionary, thanks Lin Fanshun <fundset@hotmail.com> for providing data file.
 Add DrEye4in1 dictionary, thanks Lilian <fairyland@gmail.com> for providing data file.
@@ -54,7 +54,7 @@
 
 2006.4.18
 StarDict-2.4.7 (complete) released.
-Port to GPE(http://gpe.handhelds.org).
+Port to GPE (http://gpe.handhelds.org).
 Full-text search.
 Acrobat plug-in in Win32, thanks Dewolf Xue <dewolf_matri_x@msn.com>.
 Evgeniy's patches for special character search and other changes.
@@ -94,14 +94,14 @@
 
 2005.7.20
 StarDict-2.4.5 (final) released.
-Source code optimize and many changes, thanks Evgeney <dushistov@mail.ru>.
+Source code optimize and many changes, thanks Evgeniy <dushistov@mail.ru>.
 Dutch translation and help document translation, thanks Ruud Slaats <ruudslaats@eurotechgroup.nl>.
 
 2004.11.28
 StarDict-2.4.4 (fine) released.
-sdcv-0.1(StarDict command line version) is out, developed by Evgeney <dushistov@mail.ru>.
+sdcv-0.1(StarDict command line version) is out, developed by Evgeniy <dushistov@mail.ru>.
 Try to create .idx.oft file to speed up the interpretation of the .idx file. This feature is especially useful to sdcv.
-Mueller7 optimized, thanks Evgeney.
+Mueller7 optimized, thanks Evgeniy.
 quick_eng-gre, quick_gre-eng dictionary encoding fix, thanks Christophe Combelles <ccomb@free.fr> for bug reporting.
 Add Hanzi Master dictionary, thanks Michael Robinson <robinson@netrinsics.com>.
 zh_TW help document translation and a small bug fix, thanks Roy Chan <roy.chan@debian.org.hk>.
